presto-hive-hadoop2/pom.xml

```diff
@@ -130,5 +130,25 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>test-hive-hadoop2-s3</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludedGroups>hive</excludedGroups>
```

Comment:
excluded?

Reply:
If we don't exclude here, this profile will run all the Hive tests. But if we do exclude, you can't specify both profiles to run all tests (they will counteract each other) -- you'd need a third profile.

For using a CI, having it run both (no exclude) is probably better. For just testing S3, exclude is needed. I'm fine either way. (I run them from IntelliJ)

Reply:
Is it possible to use Maven property here? If so, no need for a separate profile, just override the excluded groups list when running Maven.

Reply:
I changed the approach a bit. Not there is a default profile that excludes both `hive` and `hive-s3` group. The other profiles either exclude `hive` or `hive-s3` group. This way you can specify both profiles and both `hive` and `hive-s3` would execute.